### PR TITLE
bluetooth deprecate old params

### PR DESCRIPTION
**Pull request by tobes**
I think this should fix the deprecation stuff.

It is a bit horrible but should ensure nice backwards compatibility for all the old format options.

It is a shame it is so much code though.

### CODE REVIEW

**Comment by tobes** [comment]
ping

File: py3status/modules/bluetooth.py [hunk 1]
```diff
@@ -25,6 +26,8 @@
 
 import dbus
 
+DEFAULT_FORMAT = 'BT: [{format_device}|\?show OFF]'
```
Comment (lasers) [inline py3status/modules/bluetooth.py]: Can we change this to `BT[: {format_device}]`? It shouldn't affect users with existing config.

**Review by lasers — CHANGES_REQUESTED** [review]

File: py3status/modules/bluetooth.py (hunk 1 above)
Comment (tobes) [reply py3status/modules/bluetooth.py]: you drive a hard bargain

**Review by lasers — APPROVED** [review]

**Comment by lasers** [comment]
@tobes Good work putting everything together. I see what you did with `get_placeholders_list` there. This should work okay with the sorted list...

Although there may be no good usecase for where people actually would want to put in repeated placeholders, we should still pass them along too to keep things consistent. I just hope that day never come. You get a medal for this. Thank you. 🥇

**Pull request merged**